com.microsoft.mrtk.input/Interactors/Ray/MRTKRayInteractor.cs

```diff
@@ -163,20 +168,9 @@ public override void ProcessInteractor(XRInteractionUpdateOrder.UpdatePhase upda
                         isRelaxedBeforeSelect = false;
                     }
 
-                    if (xrController is ActionBasedController abController &&
-                            abController.rotationAction.action?.activeControl?.device is TrackedDevice device)
-                    {
-                        attachTransform.rotation = PlayspaceUtilities.ReferenceTransform.rotation * device.deviceRotation.ReadValue();
-                    }
-                    else
+                    if (devicePoseSource != null && devicePoseSource.TryGetPose(out Pose rayPose))
```

Comment:
nit: out param still named `rayPose` after the pose source was renamed

Reply:
Fixed!